### PR TITLE
add gitter link

### LINKED ISSUE #450: Add gitter link
**Actual Behaviour**
No Gitter Link to social media sectiom

**Expected Behaviour**
There should be

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by pythongiant**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://pythongiant.github.io/gci18.fossasia.org/)

- Fixes #450

## Description / Changes
Add gitter link to social media section

## Screenshots if any:

image

- - - - - - - - - - - -

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -52,7 +53,7 @@
   <link href="css/gTranslator.css" rel="stylesheet" type="text/css" media="all" />
   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css"
     media="all">
-  <link href='http://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600'
+    <link href='http://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600'
```
Comment (nikit19) [inline _includes/header.html]: incorrect indentation

Comment (sk9331657) [inline _includes/header.html]: @pythongiant  Don't change the indentation.

Comment (pythongiant) [reply _includes/header.html]: done

**Comment by pythongiant** [comment]
@sk9331657 fixed!

**Comment by pythongiant** [comment]
@saarthakchats could you point it out? LGTM on my screen

**Pull request closed without merging**